docs/reference-guides/rancher-webhook.md

```diff
@@ -17,11 +17,11 @@ Each Rancher version is designed to be compatible with a single version of the w
 
 <!-- releaseTask -->
 
-| Rancher Version | Webhook Version |
-|-----------------|:---------------:|
-| v2.8.0          |     v0.4.2      |
-| v2.8.1          |     v0.4.2      |
-| v2.8.2          |     v0.4.2      |
+| Rancher Version | Webhook Version | Prime   | Community |
+|-----------------|-----------------|---------|-----------|
+| v2.8.2          |     v0.4.2      | &check; | &check;   |
+| v2.8.1          |     v0.4.2      | &check; | &check;   |
+| v2.8.0          |     v0.4.2      |         | &check;   |
```

Comment:
I think we should follow https://ranchermanager.docs.rancher.com/versions and fill the empty cells with text (N/A, Not supported, etc.) or a symbol (X). Otherwise, the empty cells might seem like a mistake, or like we may add availability in the future.

Reply:
We might also want to include notes (in the first column or as footnotes) to explain why something isn't available (unstable release, Prime exclusive, etc)

Reply:
Added crosses (x) instead of empty cells in 53357bc7f06fbb67d99f5be7c3a51c6cd866e090.

> explain why something isn't available

I didn't add this since there's no direct relation with the webhook.